package.json

```diff
@@ -120,6 +120,7 @@
   },
   "husky": {
     "hooks": {
+      "pre-commit": "lint-staged && yarn test && yarn test:ci",
```

Comment:
I don't think that we need to run `yarn test && yarn test:ci`. `yarn test:ci` should be sufficient